PublicSDKSwiftAPITester/PurchasesErrorUtilsAPI.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PurchasesErrorUtilsAPI.swift
+//
+//  Created by Madeline Beyl on 8/25/21.
+
+import Foundation
+import Purchases
+
+func checkPurchasesErrorUtilsAPI() {
```

Comment:
this particular class is one that we might make internal, since it doesn't really add any value to customers, and it was probably an oversight to have it public in the first place